src/main/java/com/amazon/ion/impl/IonReaderContinuableCoreBinary.java

```diff
@@ -1187,7 +1187,7 @@ private static double readFloat16(ByteBuffer byteBuffer) {
         int fraction = bits & FLOAT_16_FRACTION_MASK;
         if (exponent == 0) {
             if (fraction == 0) {
-                return sign == 0 ? -0e0 : 0e0;
+                return sign == 0 ? 0e0 : -0e0;
```

Comment:
🗺️ I'm pretty sure that if I'm understanding it right, when the sign bit is `0`, the float is positive. Either way, I needed to make this change to get the Ion 1.1 round trip tests to pass.